packages/orbit-components/src/Tabs/Tabs.ct.tsx

```diff
@@ -49,4 +49,11 @@ test.describe("visual Tabs", () => {
     const component = await mount(<TabsComponentStory fullWidth />);
     await expect(component).toHaveScreenshot();
   });
+
+  test("screenshot for default with one selected", async ({ mount }) => {
```

Comment:
I'd say we can avoid extra screenshots and add a new Tab to the already existing story. I'd also say we can skip the click test. WDYT?